<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="cabecera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="nombre">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chocolate Cookies</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GphicsCal</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Graficadora con Matplotlib</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="app">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 219, 178);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="boton1">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>150</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(27.1, 52.2, 53.3);</string>
     </property>
     <property name="text">
      <string>Gr치fica de Primer Grado</string>
     </property>
    </widget>
    <widget class="QLabel" name="descApp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>139</red>
           <green>139</green>
           <blue>141</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>235</red>
           <green>219</green>
           <blue>178</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Seleccione una opci칩n</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="boton2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>180</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(27.1, 52.2, 53.3);</string>
     </property>
     <property name="text">
      <string>Gr치fica de Segundo Grado</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>210</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(27.1, 52.2, 53.3);</string>
     </property>
     <property name="text">
      <string>Gr치fica Libre</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="app3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>-110</y>
      <width>291</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(159, 197, 160);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="app2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-60</y>
      <width>291</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 192, 157);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>139</red>
           <green>139</green>
           <blue>141</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>206</red>
           <green>192</green>
           <blue>157</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>BarQ</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>app3</zorder>
   <zorder>app2</zorder>
   <zorder>app</zorder>
   <zorder>cabecera</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
